topdown/crypto_test.go

```diff
@@ -647,3 +654,91 @@ xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 		})
 	}
 }
+
+func TestExtractX509VerifyOptions(t *testing.T) {
+	tests := []struct {
+		jsonOption      *ast.Term
+		expectErr       bool
+		expectVerifyOpt x509.VerifyOptions
+	}{
+		{
+			jsonOption: ast.MustParseTerm(`{"DNSName": 1}`),
+			expectErr:  true,
```

Comment:
It would be nice to know what the error was in such cases, perhaps checking a string or something in the err.Error()